gcloud-java-bigquery/src/test/java/com/google/gcloud/bigquery/ITBigQueryTest.java

```diff
@@ -505,7 +505,12 @@ public void testInsertAllWithSuffix() {
     assertFalse(response.hasErrors());
     assertEquals(0, response.insertErrors().size());
     String newTableName = tableName + "_suffix";
-    assertNotNull(bigquery.getTable(DATASET, newTableName, TableOption.fields()));
+    BaseTableInfo suffixTable = bigquery.getTable(DATASET, newTableName, TableOption.fields());
+    while (suffixTable == null) {
+      Thread.sleep(1000L);
+      suffixTable = bigquery.getTable(DATASET, newTableName, TableOption.fields());
+    }
+    assertNotNull(suffixTable);
```

Comment:
I left it just to remember what is the expected result if we ever remove the loop. If you think redundancy is evil (which in general is) I am happy to remove.